modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4626,13 +4626,13 @@ enum QuasarSpectrum {
 
 type Query {
   # All science targets (if any) for the given observation (or environment). Set `includeDeleted` to true to include any deleted targets in the asterism.
-  asterism(
-    # Observation ID
-    observationId: ObservationId!
+  # asterism(
+  #   # Observation ID
+  #   observationId: ObservationId!
 
-    # Set to true to include deleted values
-    includeDeleted: Boolean! = false
-  ): [Target!]!
```

Comment:
I suppose you can ask for an observation's targets via `asterismGroup` with the observation set to equal a particular id but it seems a bit awkward.  I would think we should have this one eventually.